runtime/doc/eval.txt

```diff
@@ -4703,6 +4703,7 @@ input({opts})
 		cancelreturn  ""       Same as {cancelreturn} from
 		                       |inputdialog()|. Also works with 
 		                       input().
+		highlight     nothing  Highlight handler: |Funcref|.
```

Comment:
`on_input` is not right, it is currently called on redraw. And I do not like the hook or event view of the function, in my view it is more like a syntax definition, in a form of the VimL code: this allows things like caching in case of double highlight I implemented, additionally I don’t know command-line redrawing system enough to answer questions like “how many times and for which states highlight will be called when triggering `a` mapping defined like `cnoremap a bcd<C-\>e"efg"<CR>`” without experiments.